tests/integration/test_diffusion2d.py

```diff
@@ -2,18 +2,55 @@
 Tests for functionality checks in class SolveDiffusion2D
 """
 
-from diffusion2d import SolveDiffusion2D
+import numpy as np
+from ...diffusion2d import SolveDiffusion2D
```

Comment:
```suggestion
from diffusion2d import SolveDiffusion2D
```
Relative import did not work on my machine when I tried to run the tests. Nothing to worry, just fyi.